sklearn/utils/testing.py

```diff
@@ -882,3 +882,128 @@ def check_docstring_parameters(func, doc=None, ignore=None, class_name=None):
             if n1 != n2:
                 incorrect += [func_name + ' ' + n1 + ' != ' + n2]
     return incorrect
+
+
+def check_data(doc_list, type_dict, type_name, object_name, include, exclude):
+    for name, type_definition, description in doc_list:
+        # remove all whitespaces
+        type_definition = type_definition.replace(' ', '')
+        description = [element.replace(' ', '') for element in description]
+
+        if name in type_dict:
+            u_dict = type_dict[name]
+            if (u_dict['type_definition'] != type_definition or
```

Comment:
Use plain old
```python
assert actual == expected, msg
```

This way pytest can help with more or less verbosity. Msg might be "type for parameter random_state in SVC differs from in SVR".